tribits/core/package_arch/TribitsReadDepsFilesCreateDepsGraph.cmake

```diff
@@ -660,13 +676,19 @@ endmacro()
 #
 # See `Function call tree for constructing package dependency graph`_.
 #
-function(tribits_append_forward_dep_packages  packageName  inputListType)
+macro(tribits_append_forward_dep_packages  packageName  libOrTest  requiredOrOptional)
```

Comment:
Again, careful with `macro`, might be better to use `set(... PARENT_SCOPE)`.

Reply:
I made the needed adjustments.  See discussion [above](https://github.com/TriBITSPub/TriBITS/pull/530#discussion_r996158789).